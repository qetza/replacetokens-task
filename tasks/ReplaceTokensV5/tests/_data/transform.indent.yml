noargs: |
  #{indent(var1)}#

size: |
    #{indent(var1, 4)}#

firstline: |
#{indent(var1, 4, true)}#