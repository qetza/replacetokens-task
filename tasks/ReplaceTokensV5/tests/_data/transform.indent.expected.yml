noargs: |
  line1
  line2
  line3

size: |
    line1
    line2
    line3

firstline: |
    line1
    line2
    line3